src/mozilla_sec_eia/library/mlflow/__init__.py

```diff
@@ -12,6 +16,22 @@
 )
 
 
+def pyfunc_model_asset_factory(name: str, mlflow_run_uri: str):
```

Comment:
This function will create an asset to load a model from mlflow. Using `create_model` is a little bit of a weird way to provide configuration to the asset, but this ensures that the default value for `mlflow_run_uri` will show up in the dagster UI.